src/index.js

```diff
@@ -0,0 +1,182 @@
+					</legend>
+					
+					<div className="hm-post-repeat__choice">
+						<input
+							type="radio"
+							name={ `hm-post-repeat__setting-no` }
+							value=""
+							className="hm-post-repeat__dialog-radio"
+						/>
+						<label
+							htmlFor={ `hm-post-repeat-no-` }
+							className="hm-post-repeat__dialog-label"
+						>
+							No
+						</label>
+						<p>
+								No repeat selected
+						</p>
+					</div>
+					<div className="hm-post-repeat__choice">
+						<input
+							type="radio"
+							name={ `hm-post-repeat__setting-` }
+							value=""
+							className="hm-post-repeat__dialog-radio"
+						/>
+						<label
+							htmlFor={ `hm-post-repeat-no-` }
+							className="hm-post-repeat__dialog-label"
+						>
+							Daily
+						</label>
+						<p>
+								Information description
+						</p>
+					</div>
+					<div className="hm-post-repeat__choice">
+						<input
+							type="radio"
+							name={ `hm-post-repeat__setting-` }
+							value=""
+							className="hm-post-repeat__dialog-radio"
+						/>
+						<label
+							htmlFor={ `hm-post-repeat-no-` }
+							className="hm-post-repeat__dialog-label"
+						>
+							Weekly
+						</label>
+						<p>
+								Information description
+						</p>
+					</div>
+					<div className="hm-post-repeat__choice">
+						<input
+							type="radio"
+							name={ `hm-post-repeat__setting-` }
+							value=""
+							className="hm-post-repeat__dialog-radio"
+						/>
+						<label
+							htmlFor={ `hm-post-repeat-no-` }
+							className="hm-post-repeat__dialog-label"
+						>
+							Fortnightly
+						</label>
+						<p>
+								Every 2 weeks
+						</p>
+					</div>
+					<div className="hm-post-repeat__choice">
+						<input
+							type="radio"
+							name={ `hm-post-repeat__setting-` }
+							value=""
+							className="hm-post-repeat__dialog-radio"
+						/>
+						<label
+							htmlFor={ `hm-post-repeat-no-` }
+							className="hm-post-repeat__dialog-label"
+						>
+							Monthly
+						</label>
+						<p>
+								Information description
+						</p>
+					</div>
+					
+				</fieldset>
+			) }
+		/>
+        </PluginPostStatusInfo>
+     </>
+    )
+  }
+})
+
+let HMPostRepeatFields = (props) => {
+	return (
+		<>
+			{props.hmpostrepeat_metafield}
+		</>
+	)
+}
+
+// Grab stored meta value of hm-post-repeat using a redux store?!
+HMPostRepeatFields: withSelect(
+  (select => {
+    return {
+      hmpostrepeat_metafield: select('core/editor').getEditedPostAttribute('meta')['hm-post-repeat']
+    };
+  })(({ hmpostrepeat_metafield, value, label, description }) => {
+	    if (!hmpostrepeat_metafield) {
+	        return (
+				value= "",
+				label= "No",
+				description= "No repeating of post."
+	    	)
+	    }
+	    if ('fortnightly' === hmpostrepeat_metafield) {
+	        return (
+				value= "",
+				label= "Fornightly JW",
+				description= "No repeating of post."
+	    	)
+	    }
+	    return (
+	    	<>something</>
+	    );
+	}) 
+)(HMPostRepeatFields);
```

Comment:
I think you've got yourself in a bit of twist here. If you break it down you're doing:

```js
HMPostRepeatFields = withSelect( ... )( props => ... )( HMPostRepeatFields )
```

So you're chaining too many things here - `withSelect()` will only add additional props to the component you pass to its callback function, so the last part `( HMPostRepeatFields )` isn't doing anything.

In addition to that where you returning the following:

```
return (
  value="",
  label="...",
);
```

This isn't really a usable return type for a React component or indeed any JS! It should be one of `null`, a string, a react component or an array of those things.

One thing that might help you refactor this is to understand that within the `withSelect()` and also `withDispatch()` higher order components you don't just have to immediately return an object of additional props in the callback. You can apply some logic too, for example:

```jsx
const MyEnhancedComponent = withSelect( select => {
  const repeat = select('core/editor').getEditedPostAttribute('meta')['hm-post-repeat'];

  switch ( repeat ) {
    case 'fortnightly': {
      return {
         value: 'fortnightly',
         label: 'Fortnightly',
         description: 'Every 2 weeks',
      }
    }
    default: {
       return {
          value: '',
          label: 'Do no repeat',
          description: '',
       }
    }
  }
} )( props => {
  return (
    <p>repeat:</p>
    <p><input type="radio" name="repeat" value="" checked={ props.value === '' } onChange={ props.onMetaFieldChange } /> do no repeat</p>
    <p><input type="radio" name="repeat" value="fortnightly" checked={ props.value === 'fortnightly' } onChange={ props.onMetaFieldChange } /> fortnightly</p>
  )
} );
```

Note the markup above is purely demonstrative, you'll want to change that. It's really just to show that you can `select` values before you return them and decide what props you want to return using logic within the `withSelect()` or `withDispatch()` callbacks.

You should probably also move your custom component to it's own file and import it, you _might_ be hitting some issues with the order in which you're declaring variables but otherwise it's just good practice.